docs/modules/reference/pages/billing/signup/subscription.adoc

```diff
@@ -0,0 +1,130 @@
+
+You can choose from a variety of available Payara Cloud plans tailored to your needs.
+Each plan offers a specific amount of xref:billing/signup/overview.adoc#_consumption_measurement[prepaid CPU time per month], a maximum instance size, and additional features like custom domain support.
+
+
+=== Trial Plan
+
+If you are new, you can start with the xref:docs:ROOT:getting-started/cloud-trial/Signup Payara Cloud.adoc[Payara Cloud Trial].
+
+The trial plan limits usage to a maximum of 4 vCPU cores.
+
+
+
+=== After Free Trial
+After your free trial ends, you must select a subscription plan to continue using the service
+
+.Payara Cloud Subscription Plans
+image::billing/signup/billing-plan-1.png[alt="Payara Cloud Plans]
+
+After making your selection and clicking btn:[Continue to Invoicing], a summary of your chosen plan appears above the invoicing details form.
+To change your plan, click btn:[Change Subscription].
+
+
+
+
+== Invoicing Details
+
+Next, fill in your invoicing details:
+
+.Invoicing details
+image::billing/signup/cloud-trial-image3.png[alt="Invoicing details form"]
+
+Contact Name and Email::
+Enter the contact details for receiving notifications and invoices.
+It defaults to your account's email; modify if needed for different invoicing.
+
+Company Name, Address, City, Zip Code, Country::
+Provide your company's invoicing information for the invoice.
+
+Tax ID Type, Tax ID::
+Optionally, if VAT-registered, enter your VAT ID.
+Our payment processor will validate it according to the given ID Type.
+Tax type updates based on the selected country.
+
+Click btn:[Continue to payment] after filling in the details.
+
+
+== Payment method
+
+Finally, authorize a payment method for automated invoice settlement.
+Clicking btn:[Confirm payment details] might redirect you to your bank for payment authorization.
+
+Your card is charged at the trial's end and then at each billing period's start.
+
+.Payment method dialog
+image::billing/signup/cloud-trial-image4.png[]
+
+== Confirmation
+
+Agree to Payara Cloud's Terms and Conditions.
+You can also choose to opt in for product updates via email to complete the process.
+
+.Confirmation
+image::billing/signup/cloud-trial-image5.png[]
+
+== Provisioning
+
+You'll then be directed to the Payara Cloud Application Management console.
+Account provisioning may take a few minutes.
+Welcome to Payara Cloud!
+
+
+== Manage Subscriptions
+
+To see and manage your existing subscriptions, click on the Billing Management menu in the left vertical menu bar.
+Through the Billing Management menu, you can xref:docs:reference:billing/subscription/user-subscription.adoc[manage users] in your subscription, view a list of your active subscriptions and xref:docs:reference:billing/signup/additional.adoc[create new subscriptions].
+
+
+.Manage Subscription
+image::billing/signup/billing-plan-2.png[alt="Payara Cloud Plans]
```

Comment:
Click to open image in new tab change is required as suggested in earlier comment